### PR TITLE
New traitor flavors

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Adds new traitor flavors:

Defector: A Nanotrasen employee bribed to achieve syndicate interests
Radicalized: a Nanotrasen employee who has been radicalized by syndicate ideology (maybe all those posters they have been hanging up)
Hired Mercenary: A high profile mercenary hired by the syndicate to get their dirty work done.

Internal affair agent has been changed to Rival Saboteur

Rival Saboteur: A member of a different rival Nanotrasen division, sent to the station to sabotage and make it look like a syndicate job. Maybe they are jealous of ss13's high funding.

Additionally:
Traitors who are not directly affiliated with the syndicate no longer get added to the battle cruiser team if they summon it.
Some traitors are now on the "maybe hijack" list, as opposed to being devided into traitors that only hijack and traitors that never hijack. Because hijack is pretty rare, this lets some of those traitors show up more often.
## Why It's Good For The Game
New flavors are fun. Internal affairs agent has some particularly bad clashes with how traitor is played, like calling cards and the battle cruiser and all that, so this is a bit of a twist on it to make it work.
## Changelog
:cl: itseasytosee
del: Non-syndicate aligned flavors will not be added to the battlecrusier team if they summon it.
spellcheck: replaced IAA with 4 other traitor flavors
/:cl:

### CODE REVIEW

**Comment by xzero314** [comment]
Removing players from being added to the battlecruiser team is weird. Its a cool picture on the end game report to see yourself on the list with the cruiser team.  It also lets them know not to kill you

**Comment by carlarctg** [comment]
it is beyond stupid for an internal affairs agent to get a syndicate battlecruiser team. the solution isnt limiting based on rng flavor its removing the dumb flavors that dont fit the game

**Comment by itseasytosee** [comment]
> it is beyond stupid for an internal affairs agent to get a syndicate battlecruiser team. the solution isnt limiting based on rng flavor its removing the dumb flavors that dont fit the game

yes thats why IAA is being removed

**Comment by carlarctg** [comment]
honestly even then thats ridiculous but it's better ig

**Comment by Sadboysuss** [comment]
"You are getting a faction based on pure rng, and because of this, we are giving you a disadvantage and barring you from participating in the final objective you worked hard to achieve called battlecruiser, no other objective has this and there is no warning or indication of it, get cucked. So now that you called the battlecruiser, go to the escape pod or something since you have nothing better to do."

File: code/__DEFINES/antagonists.dm [hunk 1]
```diff
@@ -153,46 +153,54 @@
...
 	"Bee Liberation Front",
-	"Gone Postal",
 	"Tiger Cooperative Fanatic",
-	"Waffle Corporation Terrorist",
+	"Radicalized",
 ))
 
-///employers who hire agents to do a task and escape... or martyrdom. whatever
-GLOBAL_LIST_INIT(normal_employers, list(
+///employers who will never hire agents to hyjack
```
Comment (Sadboysuss) [inline code/__DEFINES/antagonists.dm:194]: hyjack

**Comment by Exester509** [comment]
Why are you so hellbent on imposing mediocrity on a mechanic that you can simply ignore at will.
The popup could tell you that you're an evil donkey and gameplay wouldn't change at all.
In fact, your suggestions actually carry balance changes and make some randomly chosen flavors better than others (because the very geared up nukies you just summoned can kill you now). I find this extremely lazy and wouldn't want to see it merged until an actually interesting alternative has been drafted.

**Comment by DaydreamIQ** [comment]
Hired merc is literally just a more boring version of Gorlex so theres really no meed for it, the rest of the flavours feel kind of uninspired. For comparision the non syndicate options we have are literally someone who is that desperate to get a promotion that they're sabotaging everyone else, The IAA who have an ass backwardsway of working and some crackhead whos declaring themselves a champion of evil, these just feel like you're checking a box to bloat it more. Also if you're gonna make it so that certain factions can't be associated with a battlecruiser then at the very least add a preference for it otherwise you're just screwing over people who get unlucky. And as a note traitor flavours are meant to be OPTIONAL not binding, so giving them mechanical differences just muddies things even more

**Comment by xzero314** [comment]
This pr is just a blatant reskin of their previous attempt that was blocked, now with a deceptive title. I dont think your lack of creativity and imagination on what to do with these fun traitor flavours should be a cause for them to be removed. IAA is the coolest traitor flavour by far

File: code/__DEFINES/antagonists.dm [hunk 2]
```diff
@@ -153,46 +153,54 @@
 /// JSON file containing spy objectives
 #define SPY_OBJECTIVE_FILE "antagonist_flavor/spy_objective.json"
 
-///employers that are from the syndicate
+///traitors that are from the syndicate or alligned with the syndicate
```
Comment (AMyriad) [inline code/__DEFINES/antagonists.dm:156]:
```suggestion
///traitors that are from the syndicate or aligned with the syndicate
```

File: code/__DEFINES/antagonists.dm (hunk 1 above)
Comment (AMyriad) [inline code/__DEFINES/antagonists.dm:194]:
```suggestion
///employers who will never hire agents to hijack
```

File: strings/antagonist_flavor/traitor_flavor.json [hunk 3]
```diff
@@ -32,12 +32,20 @@
...
 		"introduction": "You are from Cybersun Industries.",
 		"roundend_report": "was a specialist from Cybersun Industries.",
 		"ui_theme": "syndicate",
-		"uplink": "You have been supplied the tools for the job in the form of a standard syndicate uplink."
+		"uplink": "You have been supplied the tools for the job in the form of a standard Syndicate uplink."
+	},
+	"Defector": {
+		"allies": "You are new to the secret agent business, be wary of others and look out for yourself.",
+		"goal": "The Syndicate picked the right time to offer you money. Do their dirty work for them and get paid.",
+		"introduction": "You are the Syndicate Defector.",
```
Comment (AMyriad) [inline strings/antagonist_flavor/traitor_flavor.json:45]:
```suggestion
		"introduction": "You are a Syndicate Defector.",
```

File: strings/antagonist_flavor/traitor_flavor.json [hunk 4]
```diff
@@ -79,14 +79,38 @@
 		"ui_theme": "neutral",
 		"uplink": "You've connected to the black market to clean this mess up. If there's no evidence, there's no crime."
 	},
+	"Hired Mercenary": {
+		"allies": "The Syndicate hired you, but their agents are notoriously twitchy. Use your own judgment to decide who can be trusted.",
+		"goal": "This job pays enough to finally get out of the business. Complete your work, and you'll be sipping drinks on the beach in no time.",
+		"introduction": "You are the Syndicate Hired Mercenary.",
```
Comment (AMyriad) [inline strings/antagonist_flavor/traitor_flavor.json:85]:
```suggestion
		"introduction": "You are a Syndicate Hired Mercenary.",
```

File: strings/antagonist_flavor/traitor_flavor.json [hunk 5]
```diff
@@ -79,14 +79,38 @@
...
-		"allies": "You are the only operative we are sending, any others are fake. All other syndicate operatives are not to be trusted, with the exception of Cybersun operatives.",
+		"allies": "You are the only operative we are sending, any others are fake. All other Syndicate operatives are not to be trusted, with the exception of Cybersun operatives.",
 		"goal": "Avoid killing innocent personnel at all costs. You are not here to mindlessly kill people, as that would attract too much attention and is not our goal. Avoid detection at all costs.",
 		"introduction": "You are the MI13 Agent.",
 		"roundend_report": "was an MI13 agent.",
 		"ui_theme": "syndicate",
 		"uplink": "You have been provided with a standard uplink to accomplish your task."
 	},
+	"Rival Saboteur": {
+		"allies": "Don't make your true alligences obvious. Syndicate agents could make unlikely allies if you could trick them into thinking you are one of them.",
```
Comment (AMyriad) [inline strings/antagonist_flavor/traitor_flavor.json:99]:
```suggestion
		"allies": "Don't make your true allegiances obvious. Syndicate agents could make unlikely allies if you could trick them into thinking you are one of them.",
```

File: strings/antagonist_flavor/traitor_flavor.json [hunk 6]
```diff
@@ -79,14 +79,38 @@
...
+		"goal": "Sabotage the station to make this division look bad. Make this look like a Syndicate job, you can't let it be traced back to your sector.",
+		"introduction": "You are a saboteur from a rival division of nanotrasen.",
+		"roundend_report": "was a Rival Saboteur.",
+		"ui_theme": "ntos",
+		"uplink": "You have been given a captured Syndicate uplink to get your task done."
+	},
+	"Radicalized": {
+		"allies": "Down with Nanotrasen's tyrany! Surround yourself with likeminded indivduals.",
```
Comment (AMyriad) [inline strings/antagonist_flavor/traitor_flavor.json:107]:
```suggestion
		"allies": "Down with Nanotrasen! Surround yourself with like-minded individuals.",
```

**Comment by itseasytosee** [comment]
Thank you for your diligent grammar checking, you are truly preforming a social service.

**Comment by moocowswag** [comment]
What if we could just pick our starting tot lore? Each with their own final objectives?

**Comment by itseasytosee** [comment]
Yeah I do like that idea. Mechanical changes would have alot more wiggle room if they are opt in. That's not to say there wouldn't be complications with a 100% opt in approach.

**Comment by itseasytosee** [comment]
@tralezab

**Pull request closed without merging**